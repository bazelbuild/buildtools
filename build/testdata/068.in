manifest = [
   # buildifier: table sort 1
   ("ab", "bc", "cd"),
   ("a",  "b",  "c"),
   ("",   "",    ""),
]