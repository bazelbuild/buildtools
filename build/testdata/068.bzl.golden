# Test sorting of call arguments.

foo(
    name = "t",
    z = 1,
    a = 0,
)

foo.bar.baz(
    name = "x",
    c = 0,
    b = 1,
    d = 2,
    a = 3,
)

baz(
    name = "y",
    c = foo.bar(
        "positional1",
        42,
        c = 0,
        b = 1,
        d = 2,
        a = 3,
    ),
)

foo.bar(
    z = 1,
    a = 2,
).bar(
    name = "z",
    c = 0,
    b = 1,
    d = 2,
    a = 3,
)
